dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -414,4 +424,50 @@ public Config getConfig() {
         }
     }
 
+    @GET
+    @Timed
+    @Path("cli-info")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Get Dockstore CLI information.", description = "Get Dockstore CLI information. NO authentication")
+    @ApiOperation(value = "Get Dockstore CLI information.", notes = "Get Dockstore CLI information. NO authentication")
+    public CLIInfo getCliVersion() {
+        GHRelease ghRelease;
+        PagedIterable<GHRelease> allReleases;
+        CLIInfo cliInfo = new CLIInfo();
+        try {
+            GitHub github = new GitHubBuilder().build();
+            GHRepository repository = github.getRepository("dockstore/dockstore-cli");
+            ghRelease = repository.getLatestRelease();
```

Comment:
Do you know if this always returns a published release?

Reply:
Yes this is the way it is done in the CLI and the [GitHub docs say ](https://docs.github.com/en/rest/releases/releases#get-the-latest-release)
_The latest release is the most recent non-prerelease, non-draft release, sorted by the created_at attribute._